src/constants.js

```diff
@@ -1,40 +1,99 @@
-export const NETWORK_TYPE = {
-  ethereum: 'ethereum'
-};
+import _ from 'lodash';
+import colors from './colors';
 
-export const NETWORK_IDS = {
-  '0': 'olympic',
-  '1': 'frontier',
-  '2': 'expanse',
-  '3': 'ropsten',
-  '4': 'rinkeby',
-  '5': 'goerli',
-  '8': 'ubiq',
-  '42': 'kovan',
-  '61': 'classic',
-  '77': 'sokol',
-  '99': 'core',
-  '7762959': 'musicoin'
-};
+export const NetworkTypes = Object.freeze({
+  ETHEREUM: 'ethereum',
+  SUBSTRATE: 'substrate'
+});
 
-export const NETWORK_ID = Object.entries(NETWORK_IDS).reduce(
-  (acc, [key, value]) => Object.assign(acc, { [value]: key }),
-  {}
-);
+export const EthereumNetworkIds = Object.freeze({
+  OLYMPIC: '0',
+  FRONTIER: '1',
+  EXPANSE: '2',
+  ROPSTEN: '3',
+  RINKEBY: '4',
+  GOERLI: '5',
+  UBIG: '8',
+  KOVAN: '42',
+  CLASSIC: '61',
+  SOKOL: '77',
+  CORE: '99',
+  MUSICOIN: '7762959'
+});
 
-export const NETWORK_LIST = ['1', '61', '3', '42', '5'];
+export const SubstrateNetworkIds = Object.freeze({
+  KUSAMA: 's0'
+});
 
-export const NETWORK_TITLES = {
-  [NETWORK_ID.frontier]: 'Ethereum',
-  [NETWORK_ID.classic]: 'Ethereum Classic',
-  [NETWORK_ID.ropsten]: 'Ropsten Testnet',
-  [NETWORK_ID.kovan]: 'Kovan Testnet',
-  [NETWORK_ID.goerli]: 'Görli Testnet'
+const substrateNetworkRaw = {
+  [SubstrateNetworkIds.KUSAMA]: {
+    name: 'kusama',
+    ss58Prefix: 2,
+    balanceModuleId: 123
+  }
 };
 
-export const NETWORK_COLOR = {
-  [NETWORK_ID.frontier]: '#977CF6',
-  [NETWORK_ID.classic]: '#FC2166'
+const ethereumNetworkRaw = {
+  [EthereumNetworkIds.OLYMPIC]: {},
+  [EthereumNetworkIds.FRONTIER]: {
+    title: 'Ethereum',
+    color: '#977CF6',
+    secondaryColor: colors.card_bg,
+    available: true
+  },
+  [EthereumNetworkIds.CLASSIC]: {
+    title: 'Ethereum Classic',
+    color: '#8C7166',
+    secondaryColor: colors.card_bg,
+    available: true
+  },
+  [EthereumNetworkIds.EXPANSE]: {
+    title: 'Expanse'
+  },
+  [EthereumNetworkIds.ROPSTEN]: {
+    title: 'Ropsten Testnet',
+    available: true
+  },
+  [EthereumNetworkIds.RINKEBY]: {
+    title: 'Rinkeby Testnet'
+  },
+  [EthereumNetworkIds.GOERLI]: {
+    title: 'Görli Testnet',
+    available: true
+  },
+  [EthereumNetworkIds.KOVAN]: {
+    title: 'Kovan Testnet',
+    available: true
+  },
+  [EthereumNetworkIds.SOKOL]: {},
+  [EthereumNetworkIds.CORE]: {},
+  [EthereumNetworkIds.MUSICOIN]: {}
 };
 
-export const DEFAULT_NETWORK_COLOR = '#F2E265';
+export const ETHEREUM_NETWORK_LIST = _.mapValues(
+  ethereumNetworkRaw,
+  (ethereumNetwork, networkId) =>
+    _.defaults(ethereumNetwork, {
+      protocol: NetworkTypes.ETHEREUM,
+      color: '#F2E265',
+      secondaryColor: colors.card_text,
+      available: false,
+      title: `Ethereum_${networkId}`,
+      ethereumChainId: networkId
```

Comment:
could you please also rename `chainId` to `ethereumChainId` everywhere where it's appropriate?

Reply:
also as @Tbaut mentioned